### PR TITLE
Readable Permission names

### PR DESCRIPTION
**Pull request by v0idst4r**
Provides readable permission names, useful for some commands in bots.

### CODE REVIEW

**Comment by MinnDevelopment** [comment]
Please make PRs to `development` not `master`.

File: src/main/java/net/dv8tion/jda/core/Permission.java [hunk 1]
```diff
@@ -23,49 +23,61 @@
...
+    Permission(int offset, boolean isGuild, boolean isChannel, String name)
     {
         this.offset = offset;
         this.isGuild = isGuild;
         this.isChannel = isChannel;
+        this.name = name;
+    }
+    
+    /**
+     * The readable name as used in the Discord client.<br>
```
Comment (Almighty-Alpaca) [inline src/main/java/net/dv8tion/jda/core/Permission.java]: the '<br>' is unneeded

**Comment by v0idst4r** [comment]
Ok

**Comment by MinnDevelopment** [comment]
This needs to be rebased onto `development` to exclude the merge commit	620e380

**Comment by v0idst4r** [comment]
@MinnDevelopment Is it possible to do that without command line git?

**Comment by v0idst4r** [comment]
Going to remake this pr since I used the wrong branch

**Pull request closed without merging**